src/pokemon_size_record.c

```diff
@@ -95,10 +93,8 @@ static u32 GetMonSize(u16 species, u16 b)
 
 static void FormatMonSizeRecord(u8 *string, u32 size)
 {
-#ifdef UNITS_IMPERIAL
     //Convert size from centimeters to inches
-    size = (f64)(size * 10) / (CM_PER_INCH * 10);
-#endif
+	size = (f64)(size * 10) / (CM_PER_INCH * 10);
```

Comment:
spaces instead of tabs

Reply:
solved with https://github.com/rh-hideout/pokeemerald-expansion/pull/4183/commits/fd0ab7664345612537c62e1e878263f02568fbfa